predicators/ground_truth_models/kitchen/nsrts.py

```diff
@@ -60,7 +60,7 @@ def get_nsrts(env_name: str, types: Dict[str, Type],
         AtPreTurnOff = predicates["AtPreTurnOff"]
         AtPrePushOnTop = predicates["AtPrePushOnTop"]
         AtPrePullKettle = predicates["AtPrePullKettle"]
-        Close = predicates["Close"]
+        Close = predicates["Closed"]
```

Comment:
whoops thought we fixed this last time! anyway can you also change `Close` to `Closed` on the left hand side?